features/bevragen/persoon/verblijfplaats/dev/autorisatie-gba.feature

```diff
@@ -0,0 +1,300 @@
+      | verblijfadres.woonplaats              | 10120 80910 81010 81110 81115 81120 81130 81140 81150 81160 81210 81310       | 81170                |
+      | verblijfadres.regel1                  | 10120 80910 81010 81110 81150 81160 81210 81310 81320 81340 81350             | 81330                |
+      | verblijfadres.regel2                  | 10120 80910 81010 81110 81150 81160 81210 81310 81320 81330 81350             | 81340                |
+      | verblijfadres.regel3                  | 10120 80910 81010 81110 81150 81160 81210 81310 81320 81330 81340             | 81350                |
+      | adresseerbaarObjectIdentificatie      | 10120 80910 81110 81115 81120 81130 81140 81150 81160 81170 81190             | 81180                |
+      | nummeraanduidingIdentificatie         | 10120 80910 81110 81115 81120 81130 81140 81150 81160 81170 81180             | 81190                |
+      | datumIngangGeldigheid                 | 10120 80910 81010 81030 81110 81150 81160 81210 81310 81320 81330 81340 81350 | 88510                |
+
+    @fout-case
+    Abstract Scenario: Afnemer vraagt om groep verblijfadres en is niet geautoriseerd voor <ontbrekende autorisatie veld>
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | <ad hoc rubrieken>              | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode | 0518   |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                           |
+      | type                | RaadpleegMetBurgerservicenummer                  |
+      | burgerservicenummer | 000000024                                        |
+      | fields              | burgerservicenummer,verblijfplaats.verblijfadres |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                                  |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3             |
+      | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden. |
+      | status   | 403                                                                     |
+      | detail   | De foutieve fields waarden zijn: fields[1].                             |
+      | code     | authorization                                                           |
+      | instance | /haalcentraal/api/brp/personen                                          |
+
+      Voorbeelden:
+      | ontbrekende autorisatie veld | ad hoc rubrieken                                                        | missende autorisatie |
```

Comment:
Ook hier is de laatste kolom niet noodzakelijk.

Reply:
aangepast naar Scenario: Afnemer vraagt om groep verblijfadres en is niet geautoriseerd voor <ontbrekende autorisatie veld> (<missende autorisatie>)